guide/groovy.md

```diff
@@ -167,13 +167,23 @@ The following API is available:
 | [FileIO](https://github.com/release-engineering/pom-manipulation-ext/blob/master/io/src/main/java/org/commonjava/maven/ext/io/FileIO.java) getFileIO() | This will return a FileIO instance for remote File resolving. |
 | [PomIO](https://github.com/release-engineering/pom-manipulation-ext/blob/master/io/src/main/java/org/commonjava/maven/ext/io/PomIO.java) getPomIO() | This will return a PomIO instance for parsing POM models. |
 
-
 This can then be invoked by e.g.
 
     pme.getBaseDir()
 
 <b>NOTE</b> : Be careful not to use <code>pme.getProperties()</code> or <code>pme.getProject().getProperties()</code> as that actually calls http://docs.groovy-lang.org/latest/html/api/org/codehaus/groovy/runtime/DefaultGroovyMethods.html#getProperties(java.lang.Object)
 
+<table bgcolor="#ffff00">
+<tr>
+<td>
+    <b>NOTE</b> : From version 4.3 the following extra API is available:
+</td>
```

Comment:
I don't think that's right - it's available now just not documented

Reply:
Didn't you need to move it in order to make it accessible? It was `protected` before, so I guess it would be accessible to any script that extended `BaseScript`, but technically wasn't part of the public API.

Reply:
Looks like: `private` in version 3.8.3, `protected` in 4.0 in `BaseScriptUtils`, but not added to `BaseScript` as `public` until now, which looks to be the file which defines the API.

Reply:
So its available from 4.0 ; can you make it that please. If you're not clear on this ping me offline to discuss.